### PR TITLE
Print git output in case of failure when installing

### PR DESCRIPTION
**Pull request by f-f**

### CODE REVIEW

File: app/Main.hs [hunk 1]
```diff
@@ -1,10 +1,11 @@
...
 
 import qualified GHC.IO.Encoding
-import qualified Turtle          as T
+import qualified System.Environment as Env
+import qualified Turtle             as T
 
 import qualified PscPackage
-import           Spago           (ModuleName(..), TargetPath(..), WithMain (..))
+import           Spago              (ModuleName (..), TargetPath (..), WithMain (..))
 import qualified Spago
```
Comment (justinwoo) [inline app/Main.hs]: jumalauta

File: app/Main.hs [hunk 2]
```diff
@@ -125,7 +126,13 @@ parser
 
 main :: IO ()
 main = do
+  -- We always want to run in UTF8 anyways
   GHC.IO.Encoding.setLocaleEncoding GHC.IO.Encoding.utf8
+  -- Stop `git` from asking for input, not gonna happen
+  -- We just fail instead. Source:
+  -- https://serverfault.com/questions/544156
+  Env.setEnv "GIT_TERMINAL_PROMPT" "0"
```
Comment (justinwoo) [inline app/Main.hs]: llol

File: app/Main.hs (hunk 1 above)
Comment (f-f) [reply app/Main.hs]: stylish-haskell 😍

File: app/Main.hs (hunk 2 above)
Comment (f-f) [reply app/Main.hs]: Bad git UX is bad

**Pull request merged**